webrtc.html

```diff
@@ -12001,8 +12001,8 @@ <h3 id="sec-constraints">Constraints</h3>
       default values for attributes that would be left uninitialized without
       the init dictionary present.</li>
       <li>InvalidMediaStreamTrackError changed to InvalidParameter.</li>
-      <li>Fire NetworkError when the data transport is closed with an
-      error.</li>
+      <li>Fire the <code><a>RTCError</a></code> event
+      when the data transport is closed with an error.</li>
```

Comment:
@aboba If I understand correctly, the `error` event for a WebSocket fires when the data channel is closed due to an error at the application level (closed before it finished being established, or buffer full), and the `close` event is used for *any* closure and provides additional protocol-level information.

In my opinion it does make sense for us to copy this pattern, since the working group already decided we should try to match WebSockets, and we have the same sort of situation here.